assets/prototype/domain/eventEditor/data/queries/datetimes/test/useInitDatetimeStatus.ts

```diff
@@ -0,0 +1,10 @@
+import { useStatus, TypeName } from '../../../../../../application/services/apollo/status';
+
+
+const useInitDatetimeStatus = (): void => {
+	const { setIsLoaded, isLoaded } = useStatus();
+	if (!isLoaded(TypeName.datetimes)) {
+		setIsLoaded(TypeName.datetimes, true);
+	}
+};
```

Comment:
It's good to have this here, not leaking this logic anywhere else.

Reply:
Yes